cla_backend/libs/eligibility_calculator/tests/test_calculator.py

```diff
@@ -931,7 +943,7 @@ def test_pensioner_300k1p_house_100k1p_mort_799999_savings(self):
             "dependants_allowance": 0,
             "employment_allowance": 0,
             "partner_employment_allowance": 0,
-            "liquid_capital": 79999,
+            "liquid_capital": 0,
```

Comment:
interesting! what's going on here?

Reply:
This is the test I had to fix - the 'assets' are not liquid capital, despite this test thinking that they are

Reply:
Feels like this bug should be moved to a separate PR, possibly against master

Reply:
As the bug is in legacy_check and doesn't affect the calculation, I'm not sure if it is worth fixing?

Reply:
Might not be worth it, but I've done a change here: https://github.com/ministryofjustice/cla_backend/pull/1066